R/MsBackendTimsTof-functions.R

```diff
@@ -77,9 +77,9 @@
   msg <- .valid_required_columns(x@indices, c("frame", "file"))
   if ("file" %in% colnames(x@indices) &&
       any(!x@indices[, "file"] %in% seq_len(length(x@fileNames))))
-    msg <- c(msg, "Some file indices are not valid")
+    msg <- c(msg, "Some file indices are not compatible with x@fileNames")
   if (any(!paste0(x@indices[, "frame"], x@indices[, "file"]) %in%
-          paste0(x@frames$Id, x@frames$file)))
+          paste0(x@frames$frameId, x@frames$file)))
     msg <- c(msg, "Some indices in x@indices are not compatible with x@frames")
```

Comment:
Same as above.

Reply:
"Some indices are out of bounds"  also here?

Reply:
Yes.